oz/sqlalchemy/middleware.py

```diff
@@ -26,9 +31,26 @@ def _sqlalchemy_on_finish(self):
         """
 
         if hasattr(self, "db_conn"):
-            if self.get_status() >= 200 and self.get_status() <= 399:
-                self.db_conn.commit()
-            else:
-                self.db_conn.rollback()
+            try:
+                if self.get_status() >= 200 and self.get_status() <= 399:
+                    self.db_conn.commit()
+                else:
+                    self.db_conn.rollback()
+            except:
+                tornado.log.app_log.warning("Error occurred during database transaction cleanup: %s", str(sys.exc_info()[0]))
```

Comment:
I think we may just need to accept that logs get polluted with potentially sensitive data - it seems unreasonable to me to attempt otherwise.

Otherwise, does sqlalchemy provide any sort of configurability for what sort of data gets leaked in a stack trace?